src/lib.rs

```diff
@@ -253,6 +254,24 @@ impl error::Error for Error {
     }
 }
 
+#[cfg(target_os = "macos")]
+mod platform {
+    use std::path::{Path, PathBuf};
+
+    pub fn ini_path(binary: &Path) -> Option<PathBuf> {
+        binary.parent().and_then(|x| x.parent()).map(|x| x.join("Resources"))
+    }
+}
+
+#[cfg(not(target_os = "macos"))]
+mod platform {
+    use std::path::{Path, PathBuf};
+
+    pub fn ini_path(binary: &Path) -> Option<PathBuf> {
+        binary.parent().map(|dir| dir.to_path_buf())
+    }
+}
```

Comment:
It’s a bit unfortunate that we have to use compile ifdefs to make this work, but I guess any mistake made here will be tested downstream when geckodriver gets tested in CI.